librz/include/rz_util/ht_inc.h

```diff
@@ -76,9 +76,9 @@
 #ifndef HT_STR_OPTION_DEFINED
 #define HT_STR_OPTION_DEFINED
 typedef enum {
-	HT_STR_DUP = 0,  ///< String is copied when inserting into HT
-	HT_STR_OWN,      ///< String ownership is transferred when inserting into HT
-	HT_STR_CONST     ///< String is treated as constant and not copied when inserting into HT
+	HT_STR_DUP = 0, ///< String is copied when inserting into HT
+	HT_STR_OWN, ///< String ownership is transferred when inserting into HT
+	HT_STR_CONST ///< String is treated as constant and not copied when inserting into HT
```

Comment:
i don't think this is correctly formatted

Reply:
https://www.doxygen.nl/manual/docblocks.html#memberdoc

Reply:
i was referring to the missing spaces.

Reply:
clang format is not happy. please run it.